src/renderer/store/playback/reducer.ts

```diff
@@ -0,0 +1,77 @@
+import { Reducer } from 'redux';
+import { Action } from '../action';
+import { ApplicationStore, initialStore } from '../sharedHelpers';
+import {
+  VIDEO_PLAYING,
+  VIDEO_SEEK,
+  VIDEO_SKIP,
+  UpdatedPlaying,
+  UpdatedTimeSeek,
+  UpdatedTimeSkip,
+} from './actions';
+
+const correctTime = (inputTime: number, maxDuration: number) => {
```

Comment:
We already have a generic `clamp` function which does exactly this.

e.g. instead of calling `correctTime(time, maxDuration)` you would call `clamp(time, 0, maxDuration)`

Reply:
Okay noted